Demo/MockVenmo/AppSwitcher.swift

```diff
@@ -14,7 +14,7 @@ class AppSwitcher {
 
         successComponents?.queryItems = [
             URLQueryItem(name: "x-source", value: "Venmo"),
-            URLQueryItem(name: "resource_id", value: "cGF5bWVudGNvbnRleHRfaGg0Y3BjMzl6cTRyZ2pjZyMxYmU5OTBiNC02YTE1LTQ2ZTQtYmNmOS1iYmIwMzY4OWMwMmI=")
+            URLQueryItem(name: "resource_id", value: "cGF5bWVudGNvbnRleHRfZGNwc3B5MmJyd2RqcjNxbiM4NjE4ZThkYi0xZDJkLTQwYjktYWJjOC0zNTVlNTk5YzliNTg=")
```

Comment:
The client token used in the demo app is the same as the one used in the tests. I ran through the Venmo flow and pulled a payment context ID from the demo app to now be used here.